### PR TITLE
Added insertion_sort [C++]

### LINKED ISSUE #691: Sorting/Insertion sort[C++]
#### This issue is about [Sorting/insertion sort](https://www.google.com/amp/s/www.geeksforgeeks.org/insertion-sort/amp/)

- [x] Issue Title is in the form `Algo/DS Name [Language]`.
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This Algo/DS is worth adding to this repository.
- [x] Assign this issue to me (I want to work on this).
	- [x] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

@iiitv/project-maintainers I would like to work on this issue

### PR DESCRIPTION
**Pull request by nlok5923**
Fixes #691 

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.
- [x] This pull request shall only be reviewed and merged once the Travis build passes. No maintainer or supporter shall be obliged to review it before this condition is met.
- [x] I have mentioned the issue number correctly (with hyperlink) in this pull request description.

output screenshots:
Screenshot (1297)

### CODE REVIEW

**Review by ishanksoni — APPROVED** [review]
LGTM

File: insertion_sort/insertion_sort.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+void display(int array[], int size)
+{
+	int i = 0;
+	for (i = 0; i < size; i++)
+		cout << array[i] << " ";
+	cout << endl;
+}
+int main()
+{
+	int array[] = {5, 4, 233, 32, 1, 4, 34, 3, 23};
```
Comment (yashshah2820) [inline insertion_sort/insertion_sort.cpp:27]: make this a user input driven implementation

Comment (yashshah2820) [inline insertion_sort/insertion_sort.cpp:27]: make this a user input driven implementation

**Comment by nlok5923** [comment]
@yashshah2820 okay sir working on it.

**Review by yashshah2820 — CHANGES_REQUESTED** [review]

**Comment by nlok5923** [comment]
@yashshah2820 done changes : 
Screenshot (1374)

**Review by yashshah2820 — APPROVED** [review]

**Pull request merged**